### ISSUE TITLE
can i upload to other host

### ISSUE THREAD

**Issue by narimant** [noam148/yii2-image-manager#46]
i have a question can i upload the image with this madules to other host?

**Comment by noam148** [comment]
Nope, that's not possible. The file can only upload on his own server. You free to choose a path location on the server where to upload.
